myuw/static/vue/components/_common/husky-experience.vue

```diff
@@ -16,9 +16,11 @@
         >
           <img :srcset="srcset" :src="src" class="img-fluid" :alt="alt">
           <div class="position-absolute h-100 w-100 myuw-huskyexp-body">
-            <a :href="expLink"
-               :label="`${articleTeaserTitle}. ${articleTeaserBody}`"
-               class="d-block h-100 px-3 py-4"
+            <a
+              v-out="'articleTeaserTitle'"
```

Comment:
Should this be `v-out="articleTeaserTitle"`?

Reply:
Yes, thanks!